### PR TITLE
Pickup activity

### PR DESCRIPTION
**Pull request by bensouthern**
the basics of the pickup activity

### CODE REVIEW

**Comment by 0xABC** [comment]
A couple things based on testing:
- when filling out the details it’s hard to go to the “next” field, if you hit enter it adds a newline. can you change the behaviour so that there is a “next” button on the keyboard?
- the submit button on the detail page does not work for me (so I cannot add any item to the pickup list)

**Comment by bensouthern** [comment]
If users have not filled in critical fields in the dialog then it will not dismiss and no item will be added.  I made that more clear for users in a recent update

File: app/src/main/java/com/packants/android/Activities/PickupActivity.java [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.OnMapReadyCallback;
+import com.google.android.gms.maps.SupportMapFragment;
+import com.google.android.gms.maps.model.LatLng;
+import com.packants.android.Fragments.TouchableMapFragment;
+import com.packants.android.R;
+import com.packants.android.Utilities.PermissionUtils;
+import com.packants.android.Utilities.TouchableWrapper;
+
+/**
```
Comment (0xABC) [inline app/src/main/java/com/packants/android/Activities/PickupActivity.java]: delete this

File: app/src/main/java/com/packants/android/Activities/PickupActivity.java [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+ * Created by packants on 08/08/16.
+ */
+public class PickupActivity extends FragmentActivity implements
+        GoogleMap.OnMyLocationButtonClickListener,
+        OnMapReadyCallback,
+        ActivityCompat.OnRequestPermissionsResultCallback,
+        GoogleApiClient.OnConnectionFailedListener, GoogleApiClient.ConnectionCallbacks {
+
+    private GoogleMap mMap;
```
Comment (0xABC) [inline app/src/main/java/com/packants/android/Activities/PickupActivity.java]: I suggest we don’t use this convention (prefix `m`) unless we are going to use it consistently.

**Pull request merged**